src/content/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform.mdx

```diff
@@ -0,0 +1,216 @@
+---
+title: Get to know the New Relic platform
+tags:
+  - New Relic 
+  - Use New Relic
+  - Get started
+translate:
+  - jp
+metaDescription: Learn about how to get around the New Relic UI and some of the most important experiences. 
+redirects:
+  - /docs/new-relic-one-overview
+  - /docs/introduction-new-relic-one
+  - /docs/project-wanda-whats-new
+  - /docs/project-wanda-early-access-future-new-relic-platform
+  - /node/19491
+  - /docs/new-relic-one/use-new-relic-one
+  - /docs/new-relic-one
+  - /docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one
+  - /docs/new-relic-one/introduction-new-relic-one
+  - /docs/new-relic-solutions/new-relic-one/introduction-new-relic-one
+---
+
+import accountSwitcherWide from 'images/accounts_screenshot-crop_account-switcher-wide.png' 
+
+import transtionGuideUiRedesignCustomize from 'images/transtion-guide_screenshot-full_ui-redesign-customize.gif'
+
+import entityExplorer from 'images/accounts_screenshot-crop_entity-explorer.png'
+
+import navigator from 'images/accounts_screenshot-crop_navigator.png'
+
+import lookout from 'images/accounts_screenshot-crop_lookout.png'
+
+import search from 'images/accounts_screenshot-crop_search.png'
+
+import chartOptions from 'images/apm_screenshot-crop_chart-options.png'
+
+import traceWaterfall from 'images/apm_screenshot-crop_trace-waterfall.png'
+
+import serviceMap from 'images/apm_screenshot-full_service-map.png'
+
+import workloadMap from 'images/workloads_screenshot-full_workload-map.png'
+
+import queriesnrqlNrqlExampleTimeseries from 'images/queries-nrql_screenshot-crop_nrql-example-timeseries.png'
+
+import accountsUserMenuLocation from 'images/accounts_screenshot-full_user-menu-location.png'
+
+
+Now that you've [created a New Relic account](https://newrelic.com/signup), and [installed some observability tools](/docs/new-relic-solutions/get-started/implementation-guide-intro), you probably want some tips for getting around the platform. 
+
+To access New Relic, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**. If you have an EU datacenter account, go to **[one.eu.newrelic.com](https://one.eu.newrelic.com)**.
+
+Here we've got a 15 minute video that will show you some of the more important New Relic experiences. Note that this video is from the point of view of a [full platform user](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type). 
```

Comment:
"Below is a 15 minute video that covers some of the more important New Relic experiences from the viewpoint of a [full platform user].